tests/functional/ext/typing/redundant_typehint_argument.py

```diff
@@ -19,3 +19,22 @@
 ANSWER_10: dict | list[int] | float | str | int | bool = 10
 #  +1: [redundant-typehint-argument]
 ANSWER_11: list[int] | dict[int] | dict[list[int]] | list[str] | list[str] = ['string']
+
+# Multiple warnings for the same repeated type
+#  +1: [redundant-typehint-argument, redundant-typehint-argument, redundant-typehint-argument]
+x: int | int | int | int
+
+# No warning for type alias
+Q = int | int
```

Comment:
```suggestion
Q = int | int  # [redundant-typehint-argument]
```
Might as well fix directly, right ?

Reply:
Right now the warning is not raised for that one, so the test would fail.

Reply:
Is #8137 hard to fix ? I supposed it required calling the current check on more node than the current visit function ?

Reply:
Yeah, current check only visits annotated assignments. This would require visiting plain assigns as well as function signature (arguments and outputs). I imagine most of the rest is already in place, but it will take a little work.